<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="authorLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="recordIDValue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>660</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default RecordID</string>
    </property>
   </widget>
   <widget class="QLabel" name="recordLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>660</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Record:</string>
    </property>
   </widget>
   <widget class="QLabel" name="historyLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sales History:</string>
    </property>
   </widget>
   <widget class="QLabel" name="historyValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>1161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default History</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lastSaleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Last Sale Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastSaleValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default Last Sale</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxDesiredLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max # Desired:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberSoldLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number Sold/</string>
    </property>
   </widget>
   <widget class="QLabel" name="toOrderLable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To Order:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxDesiredValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberSoldValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="toOrderValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>#</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextRecordButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>710</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(Ctrl-Enter)Next Record</string>
    </property>
   </widget>
   <widget class="QLabel" name="orderStatusLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Order Activity Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="orderStatusValue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default Status</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="orderStatusBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(F10)Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ForgetButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>710</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(Esc)Forget</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="centerBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ISBNLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="titleEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="authorEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ISBNEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>orderStatusBox</zorder>
   <zorder>titleLabel</zorder>
   <zorder>authorLabel</zorder>
   <zorder>recordIDValue</zorder>
   <zorder>recordLabel</zorder>
   <zorder>historyLabel</zorder>
   <zorder>historyValue</zorder>
   <zorder>lastSaleLabel</zorder>
   <zorder>lastSaleValue</zorder>
   <zorder>maxDesiredLabel</zorder>
   <zorder>numberSoldLabel</zorder>
   <zorder>toOrderLable</zorder>
   <zorder>maxDesiredValue</zorder>
   <zorder>numberSoldValue</zorder>
   <zorder>toOrderValue</zorder>
   <zorder>NextRecordButton</zorder>
   <zorder>orderStatusLabel</zorder>
   <zorder>orderStatusValue</zorder>
   <zorder>SaveButton</zorder>
   <zorder>ForgetButton</zorder>
   <zorder>centerBox</zorder>
   <zorder>ISBNLabel</zorder>
   <zorder>titleEdit</zorder>
   <zorder>authorEdit</zorder>
   <zorder>ISBNEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
